lib/shoulda/matchers/active_record/validate_uniqueness_of_matcher.rb

```diff
@@ -272,6 +307,17 @@ def matches?(subject)
 
         private
 
+        def case_description
+          case @options[:case]
+          when :sensitive
+            "case sensitive "
+          when :insensitive
+            "case insensitive "
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.